assets/yarn.lock

```diff
@@ -2300,7 +2300,7 @@ path-type@^5.0.0:
   version "1.7.2"
 
 "phoenix_html@file:../deps/phoenix_html":
-  version "3.3.1"
```

Comment:
Yarn gets the `phoenix_html` package from the Elixir `deps` directory. Currently, it's of `3.3.3` version. In other words, the change is not related to the `package.json` changes, but I thought that it would be okay to include it here.